internal/controller/kubeobjects/velero/requests.go

```diff
@@ -371,7 +371,8 @@ func getBackupSpecFromObjectsSpec(objectsSpec kubeobjects.Spec) velero.BackupSpe
 		IncludedResources:  objectsSpec.IncludedResources,
 		// exclude VRs from Backup so VRG can create them: see https://github.com/RamenDR/ramen/issues/884
 		ExcludedResources: append(objectsSpec.ExcludedResources, "volumereplications.replication.storage.openshift.io",
-			"replicationsources.volsync.backube", "replicationdestinations.volsync.backube"),
+			"replicationsources.volsync.backube", "replicationdestinations.volsync.backube",
+			"PersistentVolumeClaims", "PersistentVolumes"),
```

Comment:
This may break user depending on velero backing up pvcs and pvs that ramen does not manage.

Option 1 (using the special velero lable) is little more work, but avoids this issue.
At the point we add and remove the ramen finalizer, we can also add the special velero label.

Reply:
The exclude and include resources here are per backup(esp. backup-name) and does not effect cluster wide. And our backups are uniquely named. So, ramen manages only this backup and not the others.

Reply:
Ramen backup is used by recipes that may want to backup other pvcs and pvs that ramen is not protecting. Using the label we can ensure that velero will not backup up or restore the pvs and pvcs we manage, without affecting other pvs and pvcs.